README.md

```diff
@@ -56,19 +63,20 @@ Frame is a native web3 interface that lets you sign data, securely manage accoun
 
 **On Ubuntu:** Run `sudo apt-get install build-essential libudev-dev` **before** running the demo.
 
-### Build Apps
+#### Build Apps
   ```bash
   › npm run bundle # Create app bundle
   › npm run build # Build apps for current platform
   ```
-  
-### Connect to Frame natively
+### Usage
+
+#### Connect to Frame natively
 
 Frame exposes system-wide JSON-RPC endpoints `ws://127.0.0.1:1248` and `http://127.0.0.1:1248` that you can connect to from any app. We recommend using [eth-provider](https://github.com/floating/eth-provider) to create a connection `const provider = ethProvider('frame')` as `eth-provider` will handle any connection edge cases across browsers and environments
 
-### Frame's injected provider
+#### Connect when Frame is not natively supported
```

Comment:
thanks for the PR! I don't think this section really adds any additional information, so maybe get rid of these changes, the rest look good